polars/polars-core/src/chunked_array/temporal/datetime.rs

```diff
@@ -207,14 +207,31 @@ impl DatetimeChunked {
             .unwrap()
             .and_hms_opt(0, 0, 0)
             .unwrap();
-        let fmted = match self.time_zone() {
+        let mut fmted = String::new();
+        match self.time_zone() {
             #[cfg(feature = "timezones")]
-            Some(_) => format!(
+            Some(_) => match write!(
+                fmted,
                 "{}",
                 Utc.from_local_datetime(&dt).earliest().unwrap().format(fmt)
-            ),
-            _ => format!("{}", dt.format(fmt)),
+            ) {
```

Comment:
I think it is cleaner to write something like this:

```rust
write!().map_err(|_| PolarsError::ComputeError(..))?;
```

Reply:
right, that's better - thanks!